### DISCUSSION TITLE
[Feature] Increase visibility (or contrast) on asset upload status icon

### DISCUSSION THREAD

**Discussion by mbastida123** [immich-app/immich#4738] [Feature Request]
### The feature

The cloud icon visibility depends on the photo's miniature color since it is transparent. 

For a young, handsome guy like me this is not an issue :). But for people like my parents this icon can sometimes be difficult to see. Specially if the background is whiteish. Could it be possible to change the style of said icon? I propose different options:

- Change the color of the icon based on the status. Red --> local, Green --> local + online, Blue --> online
- Make that part non transparent --> this would cover part of the photo though
- Increase the size of the icon
- Make the difference between local, online etc. more noticeable.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Gnorrk** [comment]
I support this idea. A much needed change.
